tests/integration/organizations_test.go

```diff
@@ -86,6 +88,21 @@ func TestOrganizations(t *testing.T) {
 	}
 	assert.Equal(t, slug, *updatedOrganization.Slug)
 
+	// Update organization logo
+	assert.Nil(t, updatedOrganization.LogoURL)
+	filename := "200x200-grayscale.jpg"
+	file, err := os.Open(fmt.Sprintf("../../testdata/%s", filename))
```

Comment:
[Minor] You can use [path.Join](https://pkg.go.dev/path#Join) to form the path.

Reply:
:+1: Updated here and the `clerk/organizations_test.go` file.